ext/redhat/logrotate

```diff
@@ -1,9 +1,23 @@
-/var/log/puppetlabs/puppet.log {
+# Generic logrotate config for Red Hat systems. Works with both official
+# Fedora and RHEL packages as well as PuppetLabs distributions.
+
+/var/log/puppetlabs/masterhttp.log /var/log/puppet/masterhttp.log {
+  compress
   missingok
   notifempty
-  create 0644 puppet puppet
+  # permissions/owner must be aligned with puppet/defaults.rb
+  create 0660 puppet puppet
+}
+
+/var/log/puppetlabs/puppetd.log /var/log/puppet/puppetd.log {
```

Comment:
Well I am under impression that logrotate configuration won't hurt. Imagine situation when syslog is down and puppet falls back, this will prevent from filling the volume with logs effectively.